app/scripts/views/recovery_key_reset_password.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const Cocktail = require('cocktail');
+const FormView = require('./form');
+const Template = require('templates/recovery_key_reset_password.mustache');
+
+const View = FormView.extend({
+  className: 'recovery-key-reset_password',
+  template: Template,
+
+  beforeRender() {
+  },
+
+  setInitialContext(context) {
+    const account = this.getSignedInAccount();
+    context.set({
+      email: account.get('email')
+    });
+  },
+
+  submit() {
+    this.navigate('reset_password_verified');
```

Comment:
I suspect you'll want `reset_password_complete`... OR... to show the [user a new automatically generated recovery code](https://github.com/mozilla/fxa-auth-server/pull/2463#discussion_r193396482).

Reply:
More... The `_verified` pages are usually used by the "initiating" tab of an action, the "_complete" pages used by the verification tabs.